### PR TITLE
♻️ Rewrite Maintenance Page

### PR DESCRIPTION
**Pull request by Sigmanificient**
┆Issue is synchronized with this [Trello card](https://trello.com/c/H5jriB63) by [Unito](https://www.unito.io)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*.css
```
Comment (Sigmanificient) [inline .gitignore:1]: We dont need css files as they are generated from the scss

File: index.html [hunk 2]
```diff
@@ -1,44 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Actualis Baltic</title>
-    <link rel="shortcut icon" href="assets/icon.png" type="image/x-icon">
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
```
Comment (Sigmanificient) [inline index.html:4]: 2 indent space make less width and allow easier split edit

File: index.html [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
-        <a href="https://www.facebook.com/actualisbaltic/" target="blank">
-            <img class="bubble" src="assets/facebook.svg" alt="facebook">
-            <p>actualisbaltic</p>
-        </a>
-        <a href="https://www.linkedin.com/company/actualis-baltic-eu/" target="blank">
-            <img class="bubble" src="assets/linkedIn.svg" alt="linkedIn">
-            <p>Actualis Baltic</p>
-        </a>
-    </footer>
+<header class="header">
```
Comment (Sigmanificient) [inline index.html:12]: We are using class same name as they will be faster to get from the dom and will improve maintainability

File: index.html [hunk 4]
```diff
@@ -1,44 +1,44 @@
...
+  <img class="header-img" src="assets/logo.svg" alt="Logo">
+</header>
+<main class="main">
+  <div class="maintenance">
+    <img class="maintenance-img" src="assets/maintenance.svg" alt="Icon">
+    <p class="maintenance-text">
```
Comment (Sigmanificient) [inline index.html:18]: Note that child element uses `-[ELEMENT-NAME]`. This is the same tchnic as [BEM](http://getbem.com/naming/), howver i use a single `-` & `_`

File: index.html [hunk 5]
```diff
@@ -1,44 +1,44 @@
...
+      This website is temporarily closed.
+      The technical team is taking care of it.
+      Please come back later...
+    </p>
+  </div>
+</main>
+<footer class="footer">
+  <a class="footer-link" href="tel:+370-650-81-926">
+    <img class="footer-link-icon" src="assets/phone.svg" alt="phone">
+    <span class="footer-link-text">+370 650 81 926</span>
```
Comment (Sigmanificient) [inline index.html:28]: Use `span` in links instead of `p` to not have style reset to add

File: index.html [hunk 6]
```diff
@@ -1,44 +1,44 @@
...
-        <a href="https://www.linkedin.com/company/actualis-baltic-eu/" target="blank">
-            <img class="bubble" src="assets/linkedIn.svg" alt="linkedIn">
-            <p>Actualis Baltic</p>
-        </a>
-    </footer>
+<header class="header">
+  <img class="header-img" src="assets/logo.svg" alt="Logo">
+</header>
+<main class="main">
+  <div class="maintenance">
```
Comment (Sigmanificient) [inline index.html:16]: This div will make the excat size to maintenance, thus better for people using screen readers

File: main.scss [hunk 7]
```diff
@@ -0,0 +1,76 @@
+$header-height: 74px;
+$footer-height: 48px;
```
Comment (Sigmanificient) [inline main.scss:2]: Scss var make static element size more managable

File: main.scss [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+body {
+  margin: 0;
```
Comment (Sigmanificient) [inline main.scss:5]: reset margin

File: main.scss [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+  font-family: 'Open Sans', sans-serif;
+  font-weight: bold;
```
Comment (Sigmanificient) [inline main.scss:7]: All element will inherit font props from the body

File: main.scss [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.header {
+  background: #e8e8e8;
+  box-shadow: 5px 1px 15px 3px #E8E8E8;
+  height: 74px;
+  display: grid;
+  place-items: center;
```
Comment (Sigmanificient) [inline main.scss:15]: how to center a div, basically

File: main.scss [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+.main {
+  height: calc(100vh - 122px);
+  display: grid;
+  place-items: center;
+}
+
+.maintenance {
+  display: flex;
+  flex-direction: column;
+  gap: 2vw;
```
Comment (Sigmanificient) [inline main.scss:28]: Put gap, e.g. blank space, between element

File: main.scss [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+
+  &-img {height: 10vw}
+  &-text {
+    color: #4F75E2;
+    text-align: center;
+    font-size: clamp(10px, 1.5vw, 48px);
```
Comment (Sigmanificient) [inline main.scss:34]: clamp element to avoid weird behaviors

File: main.scss [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+  .maintenance {
+    flex-direction: row;
+    &-text {text-align: left}
+  }
+
+  .footer {
+    justify-content: center;
+    &-link-text {display: block}
+  }
+}
```
Comment (Sigmanificient) [inline main.scss:76]: Mobile first layout allow generally for smaller media queries

File: main.scss [hunk 14]
```diff
@@ -0,0 +1,76 @@
...
+  background: #e8e8e8;
+  box-shadow: 5px 1px 15px 3px #E8E8E8;
+  height: $header-height;
+  display: grid;
+  place-items: center;
+  &-img {height: 62%}
+}
+
+.main {
+  height: calc(100vh - #{$header-height + $footer-height});
```
Comment (Sigmanificient) [inline main.scss:20]: Scss interpolation allow for magic calculation that append at compile time

**Comment by ReyFey** [comment]
[♻️ Rewrite Maintenance Page](https://trello.com/c/DgQ5WwOY/176-%E2%99%BB%EF%B8%8F-rewrite-maintenance-page)

**Comment by ReyFey** [comment]
test

**Comment by ReyFey** [comment]
test2

**Comment by ReyFey** [comment]
test4

**Comment by ReyFey** [comment]
[Mise à jour de la page de maintenance](https://trello.com/c/Pj8B4SPf/169-mise-%C3%A0-jour-de-la-page-de-maintenance)

File: .gitignore (hunk 1 above)
Comment (ReyFey) [reply .gitignore:1]: I will see later to switch to scss

**Review by ReyFey — APPROVED** [review]

**Pull request merged**